### DISCUSSION TITLE
Drag & Drop - task length

### DISCUSSION THREAD

**Discussion by jb-alvarado** [kimai/kimai#3907] [Questions & Answers] [answered]
Hello,
do you know if it is possible to configure kimai in a way, that when you drop a task in calendar view to a day, that it copies the hole day?

By default, when I drag and drop a task to the calendar, the task has only one hour length. But it would be nice, if the task have already 8 hours.

We using kimai also for tracking sick and holidays, and here it would be more easy if we only have to drag and drop this kind of tasks.

**Comment by kevinpapst** [comment, marked as answer]
Not possible, but interesting feature request 👍 
Please create a new issue for that, so it doesn't get lost in Discussions.
